preflight-checks/package.json

```diff
@@ -30,6 +30,7 @@
     "command-line-args": "^5.1.1",
     "command-line-usage": "^6.1.1",
     "execa": "^4.0.0",
+    "inquirer": "^6.2.0",
```

Comment:
We've been using `enquirer` in the [vip-cli](https://github.com/Automattic/vip/blob/develop/package.json#L115) project. I wonder if we should push for using the same dependencies in all our packages? Also, it seems like enquirer uses fewer dependencies 🤔 

What do you think @vaurdan @chriszarate ?

NPM links for reference: [inquirer](https://www.npmjs.com/package/inquirer) | [enquirer](https://www.npmjs.com/package/enquirer)

Reply:
Standardizing sounds good, as does fewer dependencies :+1:

Reply:
Should be straight forward to use enquirer instead. I will do the change, thanks for the heads-up! :)